crates/cheatnet/src/cheatcodes/deploy.rs

```diff
@@ -52,72 +48,56 @@ pub fn deploy_at(
         }
     }
 
-    let execute_calldata = create_execute_calldata(
-        calldata,
-        class_hash,
-        &account_address,
-        &entry_point_selector,
-        &salt,
+    let entry_point_execution_ctx = &mut EntryPointExecutionContext::new(
+        build_block_context(),
+        build_transaction_context(),
+        constants::MAX_STEPS_PER_TX,
     );
 
-    let call_result = call_contract(
-        blockifier_state,
+    let ctor_context = ConstructorContext {
+        class_hash: *class_hash,
+        code_address: Some(contract_address),
+        storage_address: contract_address,
+        caller_address: Felt252::from(0x0000_1724_9872_3497_3219_3472_1083_7402_i128)
+            .to_contract_address(), // TODO: Extract to consts
```

Comment:
👀

Reply:
Isn't it just the `TEST_ACCOUNT_CONTRACT_ADDRESS`?

Reply:
No - we don't use that anymore for anything actually i believe, but i will have to verify that statement

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/838